bot/cogs/doc.py

```diff
@@ -243,25 +305,39 @@ def __init__(self, bot: commands.Bot):
         if len(description) > 1000:
             shortened = description[:1000]
             last_paragraph_end = shortened.rfind('\n\n')
```

Comment:
```suggestion
            last_paragraph_end = shortened.rfind('\n\n', 100)
```

It might be useful to avoid cutting off at a single-sentence paragraph. Doing this resolves cases like glob cutting off at the 'Source code' beginning line if they have too large an introductory paragraph.

Reply:
resolved in 34510f52c6bbe5e2a8bbfc34f8e5d648d0d39a96